tests/__init__.py

```diff
@@ -106,7 +106,7 @@ def get_service_jwt(service_credentials, group: str = None, email=True, email_cl
     return signed_jwt
 
 
-def get_auth_header(real_header=True, authorized=True, group='dbio', email=True, email_claim=False):
+def get_auth_header(real_header=True, authorized=True, group='dbio', email=True, email_claim=True):
```

Comment:
does changing the email claim defaults have implications on other tests that use this functionality?
I would think that other sections of the codebase have to be changed to support this, unless the majority are setting this parameter when used.

Reply:
No, this should not have an implications.

This method is generating a "fake" or "real" token to be used for tests to create auth headers. The email claim field is just an extra field in the token, which ends up being an extra field in the headers.

The email claim works like the group claim, except the group claim is always added to the token (and set to the `group` kwarg value), while the email claim is set via the email from the token being used (either the real token or a fake one).